--- libopensync-plugin-synce-0.22/src/synce_plugin.c	2009-01-14 02:27:11.000000000 +0200
+++ libopensync-plugin-synce-0.22/src/synce_plugin.c.new	2009-01-14 01:59:36.000000000 +0200
@@ -205,7 +205,10 @@
 	
 		osync_debug("SynCE-SYNC", 4, "got object type: contact ids: %08x data_size: %i",ids[i],data_size);
 	
-		rra_contact_to_vcard(RRA_CONTACT_ID_UNKNOWN,data,data_size,&vcard,RRA_CONTACT_VERSION_3_0);
+		if (!rra_contact_to_vcard(RRA_CONTACT_ID_UNKNOWN,data,data_size,&vcard,RRA_CONTACT_VERSION_3_0)) {
+		  osync_context_report_error(ctx, 1, "contact_to_vcard fail. id=%i",ids[i]);			
+		  continue;
+		}
 		
 		OSyncChange *change = osync_change_new();
 		osync_change_set_member(change, env->member);
