diff -ur dbus-cxx-orig/dbus-cxx/objectproxy.cpp dbus-cxx/dbus-cxx/objectproxy.cpp
--- dbus-cxx-orig/dbus-cxx/objectproxy.cpp	2010-06-22 02:40:19.129537530 +0300
+++ dbus-cxx/dbus-cxx/objectproxy.cpp	2010-06-22 02:40:39.331535051 +0300
@@ -436,11 +436,11 @@
   {
     if ( not m_connection or not m_connection->is_valid() ) return ReturnMessage::const_pointer();
 
-//     if ( not call_message->expects_reply() )
-//     {
-//       m_connection->send( call_message );
-//       return ReturnMessage::const_pointer();
-//     }
+    if ( not call_message->expects_reply() )
+    {
+      m_connection->send( call_message );
+      return ReturnMessage::const_pointer();
+    }
 
     return m_connection->send_with_reply_blocking( call_message, timeout_milliseconds );
   }
